stdlib/2and3/ssl.pyi

```diff
@@ -136,6 +136,8 @@ PROTOCOL_TLSv1_1: int
 PROTOCOL_TLSv1_2: int
 if sys.version_info >= (3, 5):
```

Comment:
As @srittau mentioned, we can just remove the version bounds and make `PROTOCOL_TLS: int` unconditionally present, since we don't care about 3.0–3.4.